templates/s3_redundant/s3_redundant.py

```diff
@@ -190,6 +193,8 @@ def install(self):
             passive_data = dict(active_data)
             passive_data['master'] = active_tlog
             passive_data['namespaces'] = namespaces
+            passive_data['excludeNodesVM'] = []
```

Comment:
why on 2 lines, we can have this one line instead
```suggestion
            passive_data['excludeNodesVM'] = [active_dmvm.data['nodeId']]
```

Reply:
was expecting some comment that will add more to the list. But seems not :-D